registry_provider_platform_integration_test.go

```diff
@@ -0,0 +1,398 @@
+//go:build integration
+// +build integration
+
+package tfe
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRegistryProviderPlatformsCreate(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	provider, providerTestCleanup := createRegistryProvider(t, client, nil, PrivateRegistry)
+	defer providerTestCleanup()
+
+	version, versionCleanup := createRegistryProviderVersion(t, client, provider)
+	defer versionCleanup()
+
+	versionID := RegistryProviderVersionID{
+		RegistryProviderID: RegistryProviderID{
+			OrganizationName: provider.Organization.Name,
+			RegistryName:     provider.RegistryName,
+			Namespace:        provider.Namespace,
+			Name:             provider.Name,
+		},
+		Version: version.Version,
+	}
+
+	t.Run("with valid options", func(t *testing.T) {
+		options := RegistryProviderPlatformCreateOptions{
+			OS:       "foo",
+			Arch:     "scrimbles",
```

Comment:
ah, yes, scrimbles64

Reply:
Lol, I couldn't find the meaning of "scrimbles" in the dictionary, but this sounds funny 😆

Reply:
It's a made up word from a former coworker from long ago that I still use 🥳